ivy/utils/backend/sub_backend_handler.py

```diff
@@ -22,6 +22,8 @@
 ):
     if backend.startswith("__"):
         continue
+    if backend.endswith(".DS_Store"):
+        continue
```

Comment:
We don't need new line as a check, also it's better to create a seperate issue and PR to get this fixed, @Infrared1029 can provide more insight on a better solution than hardcoding `DS_STORE` into there.

Reply:
Sure